src/styles/blocks/catalog.scss

```
@@ -18,6 +18,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: transform $transDur, color $transDur ;
```

you want your code to be maximally clear for those who read it
```suggestion
    transition: transform $transition-duration, color $transition-duration ;
```

---

but good idea to create a variable 👍